buildtest/cli/__init__.py

```diff
@@ -312,6 +314,13 @@ def get_parent_parser():
         )
         return parent_parser
 
+    parser = argparse.ArgumentParser(
+        prog="buildtest",
+        formatter_class=argparse.RawDescriptionHelpFormatter,
+        description="buildtest is a HPC testing framework for building and running tests.",
+        usage="%(prog)s [options] [COMMANDS]",
+        epilog=epilog_str,
+    )
```

Comment:
Remove these lines. You don't need to create another `parser` object since there already is one. Simply do `parser.addparser` if that is what you need to do.

Reply:
alright

Reply:
those lines were added by mistake